tez-api/src/main/java/org/apache/tez/client/TezClientUtils.java

```diff
@@ -233,15 +230,11 @@ private static boolean addLocalResources(Configuration conf,
         } else {
           type = LocalResourceType.FILE;
         }
+      RemoteIterator<LocatedFileStatus> fileStatuses = getListFilesFileStatus(configUri, conf);
```

Comment:
getListFilesFileStatus receives a "String fileName" param, and here we pass a "configUri", can you unify and use whatever is closer to the truth? also I can see that getListFilesFileStatus creates an URI eventually, we can pass it here, right?

Reply:
done thing, the name is URI, but it isn't a URI object but string, it is extracted from a conf which has name URI, so kept the name old as configURI